### PR TITLE
Feature/convert cli es6 import

### PR DESCRIPTION
**Pull request by jzeltman**
Issue: CLI was failing in use, wasn't connected to kitchen sink

## What I did
I converted all of the files to use es6 import/export - with the exception of the index - as it is used by node, and just easier to keep as module.exports. Added @storybook/aem-cli config to kitchen sink example package.json, also added script to run sb-aem from kitchen sink. Tested the CLI, functions as expected for no-command, help, and version command. The story and package commands don't work due to the differences in folder heirarchy from the assumptions made in the package and story commands.

## How to test
From root.
- yarn
- yarn build
- stop build once successfully completed

cd examples/aem-kitchen-sink
- yarn
- yarn sb-aem
- yarn sb-aem help
- yarn sb-aem version 

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? added
- Does this need an update to the documentation? eventually

### CODE REVIEW

File: cli/src/cmds/help.ts [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+import chalk from 'chalk';
 import { log } from '../utils';
 
 const menus = {
   main: [
     `\n`,
     `Usage: ${chalk.italic('sb-aem <command> <options>')}\n`,
     `Commands:`,
-    `  init .................. Start a new project, or add to existing project`,
+    `  init .................. WIP - Start a new project, or add to existing project`,
```
Comment (CodeByAlex) [inline cli/src/cmds/help.ts]: Nice WIP flag

File: cli/src/cmds/package/index.ts [hunk 2]
```diff
@@ -4,20 +4,9 @@ import { error } from '../../utils';
...
-
-  if (Object.entries(packageJSON).length === 0) {
-    error(
-      'No package.json file found. Please run this from the directory with the package.json file for your project',
-      true
-    );
-  } else if (args.includes('install')) install(args, storybookConfig);
-  else if (args.includes('export')) exportPackage(args, storybookConfig);
+export async function Package(args, config) {
+  if (args.includes('install')) install(args, config);
```
Comment (CodeByAlex) [inline cli/src/cmds/package/index.ts]: could you make 'install' a constant?

File: cli/src/cmds/package/index.ts [hunk 3]
```diff
@@ -4,20 +4,9 @@ import { error } from '../../utils';
...
+  else if (args.includes('export')) exportPackage(args, config);
```
Comment (CodeByAlex) [inline cli/src/cmds/package/index.ts]: could you make 'export' a constant?

File: cli/src/cmds/package/index.ts [hunk 4]
```diff
@@ -32,7 +21,7 @@ module.exports = async args => {
         { title: 'Export', value: 'export' },
       ],
     });
-    if (response.operation === 'install') install(args, storybookConfig);
-    if (response.operation === 'export') exportPackage(args, storybookConfig);
+    if (response.operation === 'install') install(args, config);
```
Comment (CodeByAlex) [inline cli/src/cmds/package/index.ts]: could you make 'install' a constant?

File: cli/src/cmds/package/index.ts [hunk 5]
```diff
@@ -32,7 +21,7 @@ module.exports = async args => {
...
+    if (response.operation === 'export') exportPackage(args, config);
```
Comment (CodeByAlex) [inline cli/src/cmds/package/index.ts]: could you make or really use the previously made 'export' a constant?

File: cli/src/index.ts [hunk 6]
```diff
@@ -1,29 +1,34 @@
...
+  const config = getConfig();
 
   // We can't check the version until this package is actually published on NPM.
   // checkVersion();
 
   switch (cmd) {
     case 'story':
     case 'stories':
-      require('./cmds/stories/story')(args);
+      Story(args, config);
```
Comment (CodeByAlex) [inline cli/src/index.ts]: why are these functions caps?

File: examples/aem-kitchen-sink/package.json [hunk 7]
```diff
@@ -28,5 +29,27 @@
...
+    "relativeProjectRoot": "../../",
+    "uiApps": "./components",
+    "jcrRootPath": "./components",
+    "namespace": "storybook",
+    "componentPath": "./components",
+    "clientlibsPath": "./components",
+    "packageJSON": "./package.json",
+    "storybookLocation": "./.storybook/",
+    "storybookStoryLocation": "./components",
+    "aemContentPath": "/content/storybook/storybook-library",
```
Comment (CodeByAlex) [inline examples/aem-kitchen-sink/package.json]: would be nice to see if we could make some default assumptions

File: cli/src/cmds/package/index.ts (hunk 2 above)
Comment (jzeltman) [reply cli/src/cmds/package/index.ts]: Done

File: cli/src/cmds/package/index.ts (hunk 4 above)
Comment (jzeltman) [reply cli/src/cmds/package/index.ts]: Done

File: cli/src/index.ts (hunk 6 above)
Comment (jzeltman) [reply cli/src/index.ts]: Adding names. Then naming the package command, and it's a protected word. Then went with caps. Realize now that's a class convention. Switched to [command]Command for clartiy.

**Pull request merged**